### PR TITLE
OH GAWD WHAT AM I DOING IM GONNA CRASH THE SYSTEM HALP ME I'VE NEVER DONE THIS BEFORE.

### PR DESCRIPTION
**Pull request by chococookies131**
FIXED SOME C ERROR MESSAGES

I LIKE CAPS LOCK.

### CODE REVIEW

File: include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h
Comment (RubyNova) [inline include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h:27]: What actually changed here?

**Comment by jeuxjeux20** [comment]
Are you ok?

File: include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h
Comment (RubyNova) [inline include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h:17]: Please note, this is a UK english codebase. Initialized needs to be spelt as Initialised.

File: src/NovelRT.Interop/NrtInteropErrorHandlingInternal.cpp
Comment (RubyNova) [inline src/NovelRT.Interop/NrtInteropErrorHandlingInternal.cpp:13]: US spelling of Initialised here also needs to be changed.

**Comment by ismaelbrowne** [comment]
I THINK SO

**Review by RubyNova — CHANGES_REQUESTED** [review]
Only two minor changes and a question. Once these are addressed (and CI passes) I'll merge. :)

**Comment by ismaelbrowne** [comment]
Oop left caps lock on

File: include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h
Comment (ismaelbrowne) [reply include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h:27]: Didn't change anything intentionally

File: include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h
Comment (ismaelbrowne) [reply include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h:17]: Ok

File: src/NovelRT.Interop/NrtInteropErrorHandlingInternal.cpp
Comment (ismaelbrowne) [reply src/NovelRT.Interop/NrtInteropErrorHandlingInternal.cpp:13]: KK

**Comment by RubyNova** [comment]
This is related to #247 . Looks like this addresses the custom exception messages. The ticket may need revisiting for stdlib errors.

**Review by RubyNova — APPROVED** [review]
LGTM, just waiting for CI to pass before merge.

**Pull request merged**
